synapse/api/auth.py

```diff
@@ -810,6 +810,10 @@ def validate_macaroon(self, macaroon, type_string, verify_expiry, user_id):
         else:
             v.satisfy_general(lambda c: c.startswith("time < "))
```

Comment:
Except you can't remove the verify_expiry option because it's used in ``validate_short_term_login_token_and_get_user_id``